x-pack/packages/ml/aiops_utils/src/get_window_parameters.ts

```diff
@@ -65,3 +65,46 @@ export const getWindowParameters = (
     deviationMax: Math.round(deviationMax),
   };
 };
+
+export const getSnappedWindowParameters = (
+  d: WindowParameters,
+  snapTimestamps: number[]
+): WindowParameters => {
+  const snappedBaselineMin = snapTimestamps.reduce((pts, cts) => {
+    if (Math.abs(cts - d.baselineMin) < Math.abs(pts - d.baselineMin)) {
+      return cts;
+    }
+    return pts;
+  }, snapTimestamps[0]);
+  const baselineMaxTss = snapTimestamps.filter((ts) => ts > snappedBaselineMin);
```

Comment:
❓ what does `Tss` stand for?

Reply:
`ts` = singular timestamp, `tss` = plural timestamps — I'd be fine with not abbreviating it too, would make it more clear.

Reply:
Updated to clearer naming in 3f2a5c5034db76c0767c0c7cf5ee570214be066c